config.go

```diff
@@ -18,21 +18,31 @@ tags:
 `
 
 var SampleConfig = `name: .metadata.name
+aliases: # This are how we identify the services again during reconciliation - please make sure they are very unique
+  - '"k8s:\(.metadata.name)-\(.metadata.namespace)"'
+  - '"\(.metadata.namespace)-\(.metadata.name)"'
 description: .metadata.annotations."opslevel.com/description"
-owner: .metadata.annotations."opslevel.com/owner"
+framework: .metadata.annotations."opslevel.com/framework"
+language: .metadata.annotations."opslevel.com/language"
 lifecycle: .metadata.annotations."opslevel.com/lifecycle"
-tier: .metadata.annotations."opslevel.com/tier"
+owner: .metadata.annotations."opslevel.com/owner"
 product: .metadata.annotations."opslevel.com/product"
-language: .metadata.annotations."opslevel.com/language"
-framework: .metadata.annotations."opslevel.com/framework"
+properties:
+  assign:
+    # find annotations with format: opslevel.com/properties.<property-definition name>: <value>
+    - '.metadata.annotations | to_entries |  map(select(.key | startswith("opslevel.com/property"))) | map({(.key | split(".")[2]): .value})'
```

Comment:
2 things:
1.) we want to mirror how opslevel.yml is defining this
2.) we'll need to support indiviual key:value not just maps

I've adjusted the example below to account for this.
```suggestion
properties:
    - my-prop-def-alias: '.metadata.annotations.my-prop-value'
    # find annotations with format: opslevel.com/properties.<property-definition name>: <value>
    - '.metadata.annotations | to_entries |  map(select(.key | startswith("opslevel.com/property"))) | map({(.key | split(".")[2]): .value})'
```

Reply:
@rocktavious I'll re-do this - I just realized I confused service owners vs property assignment owners. That caused me to use a different schema.

Reply:
@taimoorgit - FWIW i think the example you have to bulk getting properties is a good one but it would be k8s data like
```
metadata:
  annotations:
    opslevel.com/property/my-prop-def-alias: "my-value"
    opslevel.com/property/my-prop-def-2-alias: "200"
```

^ this is what someone could do on their k8s resources and the stuff you have here would parse that perfectly!  Just not into PropertyInput but into `map[string]opslevel.JsonString`